### PR TITLE
reporting databases

### PR DESCRIPTION
**Pull request by snopoke**
buddy @kaapstorm

### CODE REVIEW

File: ansible/vars/dev/dev_public.yml [hunk 1]
```diff
@@ -131,7 +131,8 @@ formplayer_db_name: formplayer
 #    shards: [0, 99] # pl_proxy shards assigned to this DB (total accross all DB's must be power of 2)
 
 postgresql_dbs:
-  - name: "{{ localsettings.UCR_DATABASE_NAME }}"
+  - name: commcare_ucr
```
Comment (kaapstorm) [inline ansible/vars/dev/dev_public.yml]: Not `commcarehq_ucr`?

File: ansible/roles/postgresql/tasks/main.yml [hunk 2]
```diff
@@ -139,7 +139,7 @@
     lc_ctype: 'en_US.UTF-8'
     template: 'template0'
   with_items: "{{ postgresql_dbs }}"
-  when: (item.get('host', localsettings.PG_DATABASE_HOST) == inventory_hostname) or is_monolith|bool
+  when: item.create|deafult(True) and (item.get('host', localsettings.PG_DATABASE_HOST) == inventory_hostname) or is_monolith|bool)
```
Comment (kaapstorm) [inline ansible/roles/postgresql/tasks/main.yml]: `default`

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 3]
```diff
@@ -48,19 +50,7 @@ PARTITION_DATABASE_CONFIG = {
...
-{% if localsettings.get('UCR_DATABASE_NAME') %}
-UCR_DATABASE_URL = "postgresql://{{ localsettings.PG_DATABASE_USER }}:{{ localsettings.PG_DATABASE_PASSWORD }}@{{ localsettings.PG_DATABASE_HOST }}:{{localsettings.get('PG_DATABASE_PORT')}}/{{ localsettings.UCR_DATABASE_NAME }}"
-{% endif %}
-
-{% if localsettings.get('ICDS_UCR_DATABASE_ALIAS') %}
-ICDS_UCR_DATABASE_ALIAS = "{{ localsettings.ICDS_UCR_DATABASE_ALIAS }}"
-{% endif %}
-
-{% if localsettings.get('ICDS_UCR_TEST_DATABASE_ALIAS') %}
-ICDS_UCR_TEST_DATABASE_ALIAS = "{{ localsettings.ICDS_UCR_TEST_DATABASE_ALIAS }}"
```
Comment (emord) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: @snopoke not having htis is causing dashboard errors https://sentry.io/dimagi/commcarehq/issues/337569108/?referrer=slack. Is it due to not having a new code deploy?

**Comment by emord** [comment]
Added a commit to fix dashboards on icds

File: ansible/roles/commcarehq/templates/localsettings.py.j2 (hunk 3 above)
Comment (snopoke) [reply ansible/roles/commcarehq/templates/localsettings.py.j2]: sorry about that, thanks for the fix

**Pull request merged**